File: Source/Rio.Web/src/app/shared/models/water-year-month-dto.ts
import { WaterYearDto } from "./water-year-dto";


export class WaterYearMonthDto {
    WaterYearMonthID: number;
    WaterYear: WaterYearDto;
    Month: number;
    FinalizeDate: Date;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Source/Rio.Models/DataTransferObjects/WaterYearMonthDto.cs
using System;

namespace Rio.Models.DataTransferObjects
{
    public class WaterYearMonthDto
    {
        public int WaterYearMonthID { get; set; }
        public WaterYearDto WaterYear { get; set; }
        public int Month { get; set; }
        public DateTime? FinalizeDate { get; set; }
    }
}